build.gradle

```diff
@@ -68,3 +68,4 @@ task clean(type: Delete) {
 }
 
 apply from: "${rootDir}/gradle/sonarqube.gradle"
+apply from: "${rootDir}/gradle/dependencies-graph.gradle"
```

Comment:
This is not needed. This is why we were getting this https://github.com/mapbox/mapbox-java/pull/791#issuecomment-381215309 weird graph.